examples/structured_profilers.ipynb

```diff
@@ -245,7 +245,7 @@
     "\n",
     "Below, let's remove the histogram and increase the number of samples to the labeler component (1,000 samples). \n",
     "\n",
-    "Full list of options in the Profiler section of the [DataProfiler documentation](https://capitalone.github.io/DataProfiler)."
+    "Full list of options in the Profiler section of the [DataProfiler documentation](https://capitalone.github.io/DataProfiler/docs/0.10.2/html/profiler.html#profile-options)."
```

Comment:
what happens when we update the package to a new version?

Reply:
this is very brittle in my opinion

Reply:
just tested and still works with 0.10.1 @jacob-buehler @taylorfturner

Reply:
....ah I see what u mean, breaks as in, it'll go stale

Reply:
exactly -- goes stale, yes. Not that it breaks in the sense it doesn't go that versions link, but should go to the most up to date / max(version) of the docs